pep-0646.rst

```diff
@@ -1167,6 +1167,54 @@ Footnotes
    shape begins with 'time × batch', then ``videos_batch[1][0]`` would select the
    same frame.
 
+Endorsements
+============
+
+This purpose of this PEP is to make life easier for the numerical computing
+community. How likely is it that numerical computing libraries will actually
+make use of the features proposed in this PEP?
```

Comment:
That's true, but if the steering council _are_ sceptical, I think this argument won't sway them: it would be strange for a language to adopt a controversial language feature only to support community stubs. I think the weight of this section depends on big library authors themselves making use of what we propose (even if it's in stubs that they provide rather than the actual implementation, considering that the implementation is mostly not Python).